libs/sdk-ui-tests/stories/visual-regression/kit/Overlay/Overlay.tsx

```diff
@@ -0,0 +1,214 @@
+// (C) 2007-2020 GoodData Corporation
+import React from "react";
+import { injectIntl, FormattedMessage, IntlShape, IntlProvider } from "react-intl";
+import MediaQuery from "react-responsive";
+import bindAll from "lodash/bindAll";
+import { storiesOf } from "@storybook/react";
+import { UiKit } from "../../../_infra/storyGroups";
+import { Button, Overlay, FullScreenOverlay } from "@gooddata/sdk-ui-kit";
+
+import "../styles/goodstrap.scss";
+
+/**
+ * @internal
+ */
+interface IOverlayExampleProps {
+    index: number;
+    intl: IntlShape;
+}
+interface IOverlayExampleState {
+    isOpen: boolean;
+}
+class OverlayExample extends React.Component<IOverlayExampleProps, IOverlayExampleState> {
+    static defaultProps = {
+        index: 0,
+    };
+    constructor(props: IOverlayExampleProps) {
+        super(props);
+        bindAll(this, "toggleOverlay");
```

Comment:
Please use arrow functions where possible instead of bind (I konw it is not your code, but the fix here should be no problem).

Reply:
Replaced `bindAll` with arrow function